influxdb/client.py

```diff
@@ -62,11 +62,26 @@ class InfluxDBClient(object):
     :type udp_port: int
     :param proxies: HTTP(S) proxy to use for Requests, defaults to {}
     :type proxies: dict
+    :param uri: uri to use in URL when using a reverse proxy, defaults to ''
+        example https://myserver.domain.com/influxproxy with an Apache 
```

Comment:
I was fearing a misunderstanding between the forward proxy and reverse proxy settings... having the configuration here is not mandatory to me.